src/diffusers/schedulers/scheduling_edm_euler.py

```diff
@@ -200,7 +209,7 @@ def set_timesteps(self, num_inference_steps: int, device: Union[str, torch.devic
         self.num_inference_steps = num_inference_steps
 
```

Comment:
since we have mutiple sigma schedules now, let's spread it out here similarly to what we do for `beta_schedule`
```
if sigma_schedule == "karras"
    sigmas = self._compute_karras_sigma(...)
elif sigma_schedule == ?:
    sigmas = ...
```